tools/package_tools/configs/windows_ort.yaml

```diff
@@ -0,0 +1,15 @@
+global_config:
+  bdist_tags:
+    python_tag: "py37"
```

Comment:
The python version in my env is python3.8, but the generated whl package name is mmdeploy_python-0.5.0-py37-none-win_amd64.whl.

Reply:
Sure. I will try to read it from the environment.